src/protocols/rdp/channels/rail.c

```diff
@@ -86,6 +74,7 @@ static UINT guac_rdp_rail_complete_handshake(RailClientContext* rail) {
     };
 
     /* Send client handshake response */
+    guac_client_log(client, GUAC_LOG_DEBUG, "Sending RAIL handshake.");
```

Comment:
The DEBUG-level messages in this file seem more TRACEy than DEBUGy, no? 

Having these identical DEBUG messages appear in the log every single time the handshake is completed feels like overkill to me.

Reply:
Yeah, probably so...I'll switch them over to `TRACE`, instead.

Reply:
Fixed via rebase.